ethcore/src/miner/miner.rs

```diff
@@ -277,13 +276,19 @@ impl MinerService for Miner {
 				// give the sender max balance
 				state.sub_balance(&sender, &balance);
```

Comment:
But similar code in client did. So if the code was invalid (possible overflow?) then I guess this here is too (only difference being `pending` vs `latest/blocknumber`)